src/core/shape/vertex.js

```diff
@@ -559,6 +559,62 @@ p5.prototype.curveVertex = function(...args) {
  * </code>
  * </div>
  *
+ * @example
```

Comment:
I think this is in the `endContour()` docs, did you want this example to be in `endShape()`?

Reply:
Yeah, sorry about that. I fixed it in the latest commit.